zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -582,6 +582,13 @@
       }
       websocketMsgSrv.saveInterpreterBindings($scope.note.id, selectedSettingIds);
       console.log('Interpreter bindings %o saved', selectedSettingIds);
+
+      _.forEach($scope.note.paragraphs, function(n, key) {
+        if (n.text && !n.text.startsWith('%')) {
```

Comment:
but what I mean is how it should behavior when the `text` doesn't `startsWith` `%` because of the empty space, but still has a `%`